packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -398,11 +449,24 @@ const ReleaseDetailsBody = () => {
                     <Typography>{entry.contentType.displayName || ''}</Typography>
                   </Td>
                   <Td>
-                    <ReleaseActionOptions
-                      selected={type}
-                      handleChange={(e) => handleChangeType(e, id)}
-                      name={`release-action-${id}-type`}
-                    />
+                    {release.releasedAt ? (
+                      <Typography>
+                        {formatMessage(
+                          {
+                            id: 'content-releases.page.ReleaseDetails.table.action-published',
+                            defaultMessage:
+                              'This entry was {isPublish, select, true {published} other {unpublished}}.',
+                          },
+                          { isPublish: type === 'publish' }
+                        )}
```

Comment:
I don't see the view for "done" releases in the designs. Do you know where it is? For some reason I thought we were just going to show the buttons but disable them 🤔

Reply:
Found it on the designs! I think you just need the bold around the published unpublished. Rémi did that somewhere I'll find it.

Reply:
Yes, I was just to leave a comment, I tried a lot of ways to bold that but nothing works 😢 Please let me know if you found an example

Reply:
I have this example where a custom component is provided for the b tag, but here I don't think it should be necessary:

https://github.com/strapi/strapi/blob/523f211d2b0a060359d7d02674b39cba499796d6/packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/ConfirmBulkActionDialog/index.js#L130-L137

I'll try to find an example without custom component